mbtiles/Cargo.toml

```diff
@@ -31,14 +31,16 @@ anyhow = { workspace = true, optional = true }
 clap = { workspace = true, optional = true }
 env_logger = { workspace = true, optional = true }
 serde_yaml = { workspace = true, optional = true }
+size_format.workspace = true
```

Comment:
actually this is the wrong place -- size_format is used as part of the core mbtiles library, not just CLI. This sub-section is just for the `Bin dependencies` -- that's why they are all optional.